source/getting-started/index.md

```diff
@@ -1,52 +1,55 @@
 Getting started with Ember.js is easy. Ember.js projects are created and managed
-through our [command line build tool ember-cli](http://www.ember-cli.com/). The command line tool brings:
-
-* Modern application asset management (including combining, minifying, and versioning).
-* Built-in generators to help you create components, routes, and more.
-* A conventional project layout so approaching other developers' Ember.js applications is easy.
-* Official [JavaScript modules](http://jsmodules.io/) to keep your project organized.
-* A complete testing framework.
-* Access to a growing ecosystem of [Ember Addons](http://www.emberaddons.com/).
-
+through our command line build tool [Ember CLI](http://www.ember-cli.com/).
```

Comment:
I'd rather just remove it if #528 lands first, or have #528 remove it or change it if this lands first :-p I'll drop it though, it isn't worth debate.